docs/docs/slot-validation-actions.mdx

```diff
@@ -5,105 +5,32 @@ title: Slot Validation Actions
 ---
 
-Rasa SDK provides three base classes that you can inherit from in order to implement custom slot extraction and
-validation logic:
+You can use three different types of slot validation actions to write custom extraction and validation of slot values:
 
-## `Action` class
+## Regular Custom Action
 
-You can use this class to create a regular [custom action](./custom-actions.mdx). The base class will not prevent you
-from returning arbitrary events, however a slot validation action should only return `SlotSet` and `BotUttered` events.
-Any other event type will be filtered out by the default action [`action_extract_slots`](./default-actions.mdx#action_extract_slots).
-The name of the custom action must be specified in the `action` key of the relevant
-[custom slot mapping](./domain.mdx#custom-slot-mappings) in the domain.
+You can use the `Action` Rasa SDK class to create a regular [custom action](./custom-actions.mdx).
+The base class will not prevent you from returning arbitrary events, however a slot validation action should only return
+`SlotSet` and `BotUttered` events. Any other event type will be filtered out by the default action [`action_extract_slots`](./default-actions.mdx#action_extract_slots).
+The name of the custom action must be specified in the `action` key of the relevant [custom slot mapping](./domain.mdx#custom-slot-mappings)
+in the domain.
 Note that the action name must be listed in the domain `actions` section too.
 
-## `ValidationAction` class
+## `action_validate_slot_mappings`
 
-You can extend the `ValidationAction` class in the Rasa SDK to define custom extraction and / or validation of slots that
+You can use the `action_validate_slot_mappings` action to define custom extraction and / or validation of slots that
 can be set or updated outside of a form context.
 
-:::note
-`ValidationAction` is intended for extracting slots **outside** the context of a form.
-It will ignore extraction and validation methods for any slots that have a form specified under the slot mapping's `conditions`.
-It will not run these methods when the specified form is active nor when no form is active.
-Please extend the [`FormValidationAction`](./slot-validation-actions.mdx#formvalidationaction-class) class for applying
-custom slot mappings only within the context of a form.
-:::
-
-Note that you do not need to implement the `name` method in the custom action extending `ValidationAction`, since this
-is already provided: `action_validate_slot_mappings`. However this name must be added to the domain `actions` list.
-If you override the `name` method, the custom validation action will not run because the original name is hardcoded
-in the call that the default action `action_extract_slots` makes to the action server.
+This action should extend the Rasa SDK `ValidationAction` class or the implementation of this class in your
+custom action server.
```

Comment:
```suggestion
This action is called automatically at the end of the default action [`action_extract_slots`](./default-actions.mdx#action_extract_slots), so the name must not be changed. If you are using Rasa SDK, you should extend the Rasa SDK `ValidationAction` class. If you are using a different action server, you will need to implement an action class with equivalent functionality to the Rasa SDK class. Please see [the action server docs]() for details. 
```

I think it's best to call out the redirect explicitly because with many linked words users don't always know which is the next step to go to. You can still link the specific words though.